src/lib/documents.ts

```diff
@@ -72,3 +73,26 @@ export function isNIT(str: string): boolean {
 
   return Number(verifier) === sum;
 }
+
+/**
+ * Verifies that given Passport format is valid
+ *
+ * @param  {string} str       A string representing the passport
+ * @returns {boolean}         Validity of the given passport
+ */
+export function isPassportFormat(str: string): boolean {
+  return passportRegExp.test(str.toUpperCase());
+}
+
+/**
+ * Verifies that given Passport is valid
+ *
+ * @param  {string} str       A string representing the passport
+ * @param  {string} dui       A string representing the dui
+ * @returns {boolean}         Validity of the given passport
+ */
+export function isPassport(str: string, dui: string): boolean {
```

Comment:
Please remove the DUI verification as a requirement to validate passport considering that there are many residents that use their international passports. So I don't think it would be a good idea to create a constraint, based on this parameter.

Reply:
Done in a new PR